tests/integration/ethernet_mtu_test.py

```diff
@@ -24,7 +25,7 @@ def eth1(eth1_up):
 def eth1_with_ipv6(eth1_up):
     ifstate = eth1_up[Interface.KEY][0]
     ifstate[Interface.IPV6][InterfaceIPv6.ENABLED] = True
-    libnmstate.apply(eth1_up)
+    apply_with_description("Enable IPv6 on interface eth1", eth1_up)
```

Comment:
It is a great idea to save the description in the state. To me it seems useful for the generic schema as well, so that (for now) Nmstate will just ignore the description. Later, it can be great to be able to report this back as well, if we are keeping track of state changes like using a git commit history. Then the descriptions can be used in the git commit messages.

Reply:
Maybe let's just use the top-level description instead. Re-using the existing field seems wrong, since I consider it as the description of the interface and not the description of the state.

Reply:
Changed to use top level descriptions. Please review.